### ISSUE TITLE
Bug: "Connection ready. Took xx seconds" displays only after next step completes

### ISSUE THREAD

**Issue by jim256** [hankinsoft/SQLPro#512]
<img width="361" alt="image" src="https://user-images.githubusercontent.com/15026225/95634480-3f8b9480-0a47-11eb-8800-51dd8c5a7c14.png">

I noticed that only "Initializing connection..." showed until after the full query executed, making it appear for 12.9 seconds (the full duration) that it was taking 12 seconds to establish connection, even though that part only took .07 seconds.

Loving that these times now display, though. Very helpful!

macOS 10.15.7 (19H2)
SQLPro Studio 2020.88 (Build 110729.5)
Snowflake 4.33.1

**Comment by jim256** [comment]
Looks like this actually opens sometimes. Haven't identified a pattern yet.

**Comment by hankinsoft** [comment]
I'm going to close this as a duplicate of #491 as it's part of that feature.
